File: tugas2.js
let golongan = "B";
let jamLembur = 3;

let gajiPokok = 0;
let persenLembur = 0;
let gajiLembur = 0;
let totalGaji = 0;

if (golongan === "A") {
  gajiPokok = 5000000;
} else if (golongan === "B") {
  gajiPokok = 6500000;
} else if (golongan === "C") {
  gajiPokok = 9500000;
}

if (jamLembur === 1) {
  persenLembur = 0.30;
} else if (jamLembur === 2) {
  persenLembur = 0.32;
} else if (jamLembur === 3) {
  persenLembur = 0.34;
} else if (jamLembur === 4) {
  persenLembur = 0.36;
} else if (jamLembur >= 5) {
  persenLembur = 0.38;
}

gajiLembur = gajiPokok * persenLembur;
totalGaji = gajiPokok + gajiLembur;

console.log("Golongan Karyawan : " + golongan);
console.log("Gaji Pokok        : Rp." + gajiPokok);
console.log("Jam Lembur        : " + jamLembur + " Jam");
console.log("Gaji Lembur       : Rp." + gajiLembur);
console.log("Total Penghasilan : Rp." + totalGaji);

File: Tugas2.java
import java.util.Scanner;

public class Tugas2 {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan Golongan (A/B/C): ");
        String golongan = input.nextLine().trim().toUpperCase();

        System.out.print("Masukkan Jam Lembur: ");
        int jamLembur = input.nextInt();

        input.close();

        // Gaji pokok berdasarkan golongan
        double gajiPokok = 0;
        if (golongan.equals("A")) {
            gajiPokok = 5000000;
        } else if (golongan.equals("B")) {
            gajiPokok = 6500000;
        } else if (golongan.equals("C")) {
            gajiPokok = 9500000;
        } else {
            System.out.println("Golongan tidak valid!");
            return;
        }

        // Persen lembur berdasarkan jam
        double persenLembur = 0;
        if (jamLembur == 1) {
            persenLembur = 0.30;
        } else if (jamLembur == 2) {
            persenLembur = 0.32;
        } else if (jamLembur == 3) {
            persenLembur = 0.34;
        } else if (jamLembur == 4) {
            persenLembur = 0.36;
        } else if (jamLembur >= 5) {
            persenLembur = 0.38;
        }

        // Hitung gaji
        double gajiLembur = gajiPokok * persenLembur;
        double totalGaji = gajiPokok + gajiLembur;

        // Output
        System.out.println("Golongan Karyawan : " + golongan);
        System.out.println("Gaji Pokok        : Rp." + gajiPokok);
        System.out.println("Jam Lembur        : " + jamLembur + " Jam");
        System.out.println("Gaji Lembur       : Rp." + gajiLembur);
        System.out.println("Total Penghasilan : Rp." + totalGaji);
    }
}